### PR TITLE
WIP Add FFT in Clojure

### PR DESCRIPTION
**Pull request by earthfail**
I know there are a lot of useless commit but I couldn't resolve the issue

### CODE REVIEW

File: chapters/algorithms/cooley_tukey/code/clojure/fft.clj [hunk 1]
```diff
@@ -0,0 +1,59 @@
+(ns fft.core
+  (:require [complex.core :as c]
+            [clojure.core.matrix :as m]))
```
Comment (jiegillet) [inline chapters/algorithms/cooley_tukey/code/clojure/fft.clj]: Do you use `m`? I couldn't see it.

**Comment by jiegillet** [comment]
Are you still alive? We might close this PR if it gets stale, which would be a pity since it's almost good to go.

**Comment by earthfail** [comment]
Yeah I just got alot of issues in real life I need to resolve. Thanks for the concern @jiegillet

**Comment by jiegillet** [comment]
I'm sorry to hear that. I'll keep the PR open, please take all the time you need.

**Pull request closed without merging**